tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Web Console Audit Logging
+Summary: Learn about how to audit operations on the TiDB Cloud Web Console.
+---
+
+# Web Console Audit Logging
+
+TiDB Cloud provides you with the audit log function for Web Console operations, which can record the history of user access details (such as user login to Web Console, cluster creation, etc.).
+
+> **Note:**
+>
+> Currently, the **audit logging** feature is experimental. The interface and output are subject to change.
+
+To assess the effectiveness of user access policies and other information security measures of your organization, it is a security best practice to conduct a periodic analysis of the DbaaS audit logs.
+
+## Specify auditing filter rules
```

Comment:
Line 71 says "Closing the DBaaS audit log feature is not supported".

Reply:
Do not conflict.
1. This refers to analyzing the content of the audit log file, such as the analyzed console status and cluster status information.
2. It is not supported to modify the filter condition configuration of the audit log, but all operations will be included in the audit log

Reply:
It is not recommended to modify the title. Currently, specifying filtering rules is not supported. This is a temporary state. A complete audit function should support the configuration of filtering rules.